unic/ucd/category/src/category.rs

```diff
@@ -80,7 +80,36 @@ pub enum GeneralCategory {
     Unassigned,
 }
 
-use self::GeneralCategory::*;
+use GeneralCategory::UppercaseLetter as Lu;
```

Comment:
I thought we landed on putting these in a `pub mod abbr_names` for each component, and `use abbr_names::*` whenever needed.

Is this a different case than `BidiClass` and others?